### ISSUE TITLE
Older cloudwatch logs don't clear after refresh 

### ISSUE THREAD

**Issue by amir-faliam** [loladotdev/lola#14]
### Describe the bug

```shell
When refreshing logs in relative time mode, older logs don't get cleared
```

### To Reproduce

How to reproduce:
1. Go to any cloudwatch log - All Log Streams
2. Select relative time frame (i.e. now - 5min)
3. View logs
4. wait 6 minutes
5. make another request
6. refresh logs

You'll see older logs still 

### Expected behaviour

```shell
It should only display the logs in the given time frame
```

### Screenshots

https://www.loom.com/share/987cef8af62e4f018eae727c3c2243b0

### Operating System

macOS

### OS Version

12.3.1 (21E258)

**Comment by aeby** [comment]
@amir-faliam Thanks for the report. We were able to reproduce this behavior and are working on a fix. The reason for this odd behavior is that we evaluate the relative time expression just once.

**Comment by livioso** [comment]
@amir-faliam Thanks again for the bug report! 😸 Should be fixed with today's release [v0.6.6](https://github.com/loladotdev/lola/blob/main/CHANGELOG.md#v066-may-31-2022). :shipit:

**Comment by amir-faliam** [comment]
amazing, you guys are awesome!
